qiita_db/analysis.py

```diff
@@ -812,7 +812,7 @@ def build_files(self,
             # doing it here cause it's computational cheaper
             all_ids = list(chain.from_iterable([
                 samps for _, samps in viewitems(samples)]))
-            rename_dup_samples = ((len(all_ids) != len(set(all_ids))) or
+            rename_dup_samples = ((len(all_ids) != len(set(all_ids))) and
```

Comment:
Just to make sure - the reason why it is not a big deal to merge in master ATM is b/c this functionality is not exposed to the GUI, correct?